### PR TITLE
Add apkeep

### PR DESCRIPTION
**Pull request by kzshantonu**
- [ x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Make this PR in Main bucket

**Comment by kzshantonu** [comment]
> Make this PR in Main bucket

Does not fit the criteria for main bucket https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket

File: bucket/apkeep.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.8.0",
+    "description": "command-line tool for downloading APK files from various sources",
```
Comment (rashil2000) [inline bucket/apkeep.json:3]:
```suggestion
    "description": "Command-line tool for downloading APK files from various sources",
```

File: bucket/apkeep.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "homepage": "https://github.com/EFForg/apkeep",
+    "license": "MIT",
+    "url": "https://github.com/EFForg/apkeep/releases/download/0.8.0/apkeep-x86_64-pc-windows-msvc.exe#/apkeep.exe",
+    "hash": "2f0fedbf4243ca700be99f8a9890acf88e7b9c7f7891f25cd246ea00c3bd802a",
```
Comment (rashil2000) [inline bucket/apkeep.json:7]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/EFForg/apkeep/releases/download/0.8.0/apkeep-x86_64-pc-windows-msvc.exe#/apkeep.exe",
            "hash": "2f0fedbf4243ca700be99f8a9890acf88e7b9c7f7891f25cd246ea00c3bd802a"
        }
    },
```

File: bucket/apkeep.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "bin": "apkeep.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url" : "https://github.com/EFForg/apkeep/releases/download/$version/apkeep-x86_64-pc-windows-msvc.exe#/apkeep.exe"
```
Comment (rashil2000) [inline bucket/apkeep.json:11]:
```suggestion
        "architecture": {
            "64bit": {
                "url" : "https://github.com/EFForg/apkeep/releases/download/$version/apkeep-x86_64-pc-windows-msvc.exe#/apkeep.exe"
            }
        }
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
Please make the requested changes

**Comment by rashil2000** [comment]
Inconsistent indentation. 

Pay close attention to the suggestions.

**Comment by snowman** [comment]
> > Make this PR in Main bucket
> 
> Does not fit the criteria for main bucket https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket

Which point(s) do you think doesn't fit? 

I also want to send a pull request, but I don't know which repos should I send.

But I come across this pull request, I think `apkeep` is a [command-line tool](https://github.com/EFForg/apkeep/blob/master/USAGE), so the `apkeep.json` should go to the `Main` bucket.

> For an app to be acceptable for the main bucket, it should be:
> 
> 1. a reasonably well-known and widely used developer tool
> 2. the latest stable version of the program
> 3. the full version i.e. not a trial version
> 4. a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)
> 5. a non-GUI tool

**Comment by rashil2000** [comment]
> a reasonably well-known tool

It is less than a year old (not much popular)

**Pull request merged**